tasks/post-to-cmr/index.js

```diff
@@ -107,7 +107,16 @@ async function publish(cmrFile, creds, bucket, stack) {
   );
 
   const xml = await getMetadata(cmrFile.filename);
-  const res = await cmr.ingestGranule(xml);
+  let res;
+  try {
+    res = await cmr.ingestGranule(xml);
+  }
+  catch (err) {
+    if (err.response.body.includes('Invalid username or password')) {
```

Comment:
This was giving a very generic "422 error" before, which didn't give you a good indication of what the problem was.

Reply:
Instead of creating a custom error here, why not pass along the CMR error with a prefix like 'CMR error: <error from CMR>'